build-image.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+npm run build 
+docker build -t jembi/api-config-importer:temp . 
```

Comment:
is the temp tag intentional?

Reply:
Updated to `latest` in [2611f54](https://github.com/jembi/api-config-importer/pull/1/commits/2611f54fc4b1737533146650c9ddbd75d6b26468)